src/content/app/genome-browser/components/browser-cog/BrowserCog.tsx

```diff
@@ -35,6 +35,7 @@ export type BrowserCogProps = {
 
 const BrowserCog = (props: BrowserCogProps) => {
   const { cogActivated, updateSelectedCog, trackId } = props;
+  const { trackTrackSettingsOpened } = useGenomeBrowserAnalytics();
```

Comment:
Two `track`s made it a bit confusing about this variable name at first 😁 Not that I have anything against it, but was wondering if we can replace the `track` prefix with something else.

Reply:
I was skeptical as well but I thought it will understood. I don't think `trace` or `monitor` will be  better fit here.

Reply:
I agree with Manoj that `trace` or `monitor` aren't better. `trackTrack` is ugly as hell, but I can't think of what to replace the first `track` with. `reportTrackSettingsOpened`? `sendTrackSettingsOpened`? I am kinda ok with `trackTrack` as well.

Reply:
`reportTrackSettingsOpened` looks promising. Why not that?

Reply:
Changed it to `reportTrackSettingsOpened`